python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -183,7 +189,9 @@ def __init__(
         self._target_max_block_size = target_max_block_size
         self._started = False
         self._in_task_submission_backpressure = False
+        self._in_task_submission_backpressure_reason = ""
         self._in_task_output_backpressure = False
+        self._in_task_output_backpressure_reason = ""
```

Comment:
Let's wrap this into new structure like:

```
self._tasks_state = TaskState(...)

class TaskState:
  submission_throttled: bool
  submission_throttled_reason: str = ...

  # ...
```

Reply:
done